test/parallel/test-buffer-fill.js

```diff
@@ -181,25 +182,32 @@ deepStrictEqualValues(genBuffer(4, [hexBufFill, 1, -1]), [0, 0, 0, 0]);
 
 
 // Check exceptions
-assert.throws(() => buf1.fill(0, -1), /^RangeError: Out of range index$/);
-assert.throws(() =>
-             buf1.fill(0, 0, buf1.length + 1),
-              /^RangeError: Out of range index$/);
-assert.throws(() => buf1.fill('', -1), /^RangeError: Out of range index$/);
-assert.throws(() =>
-             buf1.fill('', 0, buf1.length + 1),
-              /^RangeError: Out of range index$/);
-assert.throws(() =>
-             buf1.fill('a', 0, buf1.length, 'node rocks!'),
-              /^TypeError: Unknown encoding: node rocks!$/);
-assert.throws(() =>
-             buf1.fill('a', 0, 0, NaN),
-              /^TypeError: encoding must be a string$/);
-assert.throws(() =>
-             buf1.fill('a', 0, 0, null),
-              /^TypeError: encoding must be a string$/);
-assert.throws(() =>
-             buf1.fill('a', 0, 0, 'foo'), /^TypeError: Unknown encoding: foo$/);
+const errRegex =
```

Comment:
I don't think we need `errRegex` in this file. I think all instances of it can be replaced with `common.expectsError()`.